library/Icingadb/Widget/Detail/UserDetail.php

```diff
@@ -90,4 +95,68 @@ protected function assemble()
             700 => $this->createCustomVars()
         ], $this->createExtensions()));
     }
+
+    private function localizeTypes(array $types)
+    {
+        $localizedTypes = [];
+        foreach ($types as $type) {
+            switch ($type) {
+                case 'problem':
+                    $localizedTypes[] = t('Problem');
+                    break;
+                case 'ack':
+                    $localizedTypes[] = t('Acknowledgement');
+                    break;
+                case 'recovery':
+                    $localizedTypes[] = t('Recovery');
+                    break;
+                case 'downtime_start':
+                    $localizedTypes[] = t('DowntimeStart');
```

Comment:
It's a label/title, so please use general word boundaries/structure. (i.e. spaces)